scilpy/gradients/optimize_gradient_sampling.py

```diff
@@ -293,6 +297,7 @@ def compute_min_duty_cycle_bruteforce(points, shell_idx, bvals, ker_size=10,
 
 def compute_peak_power(q_scheme, ker_size=10):
     """
+    toDo: Description of peak power.
```

Comment:
I found this from Guillaume Gilbert : 

Optimize the diffusion gradient table by minimizing the maximum gradient load on any of the 3 axes over a preset temporal window.

This is the function that is minimized in compute_min_duty_cycle_bruteforce, via compute_peak_power.